compose/ui/ui-text/src/desktopTest/kotlin/androidx/compose/ui/text/DesktopParagraphTest.kt

```diff
@@ -94,10 +414,10 @@ class DesktopParagraphTest {
                 .isEqualTo(Rect(0f, 0f, fontSizeInPx, 60f))
 
             Truth.assertThat(paragraph.getBoundingBox(1))
-                .isEqualTo(Rect(fontSizeInPx, 0f, fontSizeInPx * 2.5f, 60f))
+                .isEqualTo(Rect(fontSizeInPx, 0f, fontSizeInPx * 2.5f, 50f))
```

Comment:
Thanks for fixing these tests!

By the way, it probably should be 50f in the first assert too. [The test](https://teamcity.jetbrains.com/buildConfiguration/JetBrainsPublicProjects_Compose_PublishFast_1ComposeDesktopTest/3942500?buildTab=tests&status=muted&expandedTest=build%3A%28id%3A3942500%29%2Cid%3A75671) is failing on the 94 line

Reply:
yes, I missed that. thanks!
It didn't fail for me because of `assumeTrue(isLinux)`